internal/handlers/cart.go

```diff
@@ -0,0 +1,67 @@
+package handlers
+
+import (
+	"Cart_Api_New/internal/errorsx"
+	"context"
+	"encoding/json"
+	"errors"
+	"log"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+func (h Handler) CreateCart(w http.ResponseWriter, r *http.Request) {
+	ctx, cancel := context.WithTimeout(r.Context(), time.Second*1)
```

Comment:
make 1 as const and name it like requestTimeToProcess

Reply:
const requestTimeToProcess can not repeat in package, so it's in handler